test/testUtils/mtohUtils.py

```diff
@@ -36,27 +39,44 @@ def checkForPlugin(pluginName: str):
         return False
     return True
 
-def checkForMayaUsdPlugin():
-    return checkForPlugin('mayaUsdPlugin')
+class MayaHydraBaseTestCase(unittest.TestCase, ImageDiffingTestCase):
+    '''Base class for mayaHydra unit tests.'''
 
-def checkForMtoAPlugin():
-    return checkForPlugin('mtoa')
+    DEFAULT_CAM_DIST = 24
 
-class MayaHydraBaseTestCase(unittest.TestCase):
-    '''Base class for mayaHydra unit tests without image comparison.'''
+    _inputDir = None
 
+    # Variables to be set in subclasses
     _file = None
-
-    DEFAULT_CAM_DIST = 24
+    _extraPluginsToLoad = []
 
     @classmethod
     def setUpClass(cls):
         if cls._file is None:
-            raise ValueError("Subclasses of MayaHydraBaseTestCase must "
-                             "define `_file = __file__`")
-        # Set up all tests with their own test directory to write out test-specific coverage information
-        fixturesUtils.setUpClass(cls._file, 'mayaHydra',
-                                         initializeStandalone=False)
+            raise ValueError("Subclasses of MayaHydraBaseTestCase must define "
+                             "`_file = __file__`")
+
+        inputPath = fixturesUtils.setUpClass(
+            cls._file, 'mayaHydra', initializeStandalone=False, 
+            suffix=('_' + cls.__name__))
+
+        if cls._inputDir is None:
+            inputDirName = os.path.splitext(os.path.basename(cls._file))[0]
+            inputDirName = testUtils.stripPrefix(inputDirName, 'test')
+            if not inputDirName.endswith('Test'):
+                inputDirName += 'Test'
+            cls._inputDir = os.path.join(inputPath, inputDirName)
+
+        cls._testDir = os.path.abspath('.')
+
+        if MAYAUSD_PLUGIN_NAME not in cls._extraPluginsToLoad:
```

Comment:
Nit-pick: in Maya terminology plugins that are needed for a scene are described with the `requires` command:
https://help.autodesk.com/cloudhelp/2024/ENU/Maya-Tech-Docs/Commands/requires.html
Maybe for familiarity we could call this list `_requiredPlugins`?  Just a suggestion, I'm fine with keeping what you have as well.

Reply:
Adjusted the naming, thanks!